### ISSUE TITLE
What does this project do? 

### ISSUE THREAD

**Issue by brightfuture** [drvinceknight/vainglory-analysis#1]

**Comment by drvinceknight** [comment]
Yeah sorry, I haven't really put up any road signs. This a repo with some code for a blog post I wrote here: http://vknight.org/unpeudemath/code/2015/09/05/picking-a-good-vainglory-jungler-with-game-theory-and-sagemath.html

I used Python and sagemath to scrape info about heroes and run a game theoretic analysis of choice of heroes.
